crates/teloxide/src/dispatching/dialogue/storage/redis_storage.rs

```diff
@@ -60,10 +65,12 @@ where
         ChatId(chat_id): ChatId,
     ) -> BoxFuture<'static, Result<(), Self::Error>> {
         Box::pin(async move {
+            let mut pool = self.pool.get().await?;
```

Comment:
This is called `conn` in the [documentation](https://lib.rs/crates/deadpool-redis), so let's also use `conn` as a name.